.github/ISSUE_TEMPLATE/Epic.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Epic template
+about: Template for creating epics
+title: "[EPIC]"
+labels: Epic
+assignees: ''
+
+---
+
+<!--
+An Epic should define a higher-level package of work or project iteration that, once finished, makes sense to merge in core.
+-->
+## Description ##
+<!-- The what -->
+
+## Rationale ##
+<!-- The why -->
+
+## Stakeholders ##
+<!-- The who. This could include developers, designers, accessibility folks, testers, devrel, PMs... -->
+
+## Dependencies & Other Context ##
```

Comment:
Maybe "Blockers", which are essentially the high priority items to do.

Reply:
Mmm... wouldn't that directly belong to the main category of tasks (MVP/required tasks)?